e2e/test/prerequisites/iot_config.cmd_template

```diff
@@ -1,102 +0,0 @@
-@REM Copyright (c) Microsoft. All rights reserved.
```

Comment:
I remember we have this file linked somewhere in our contribution prereq readme/ devbox setup; we'll need to remove that as well. 

So, this file was meant to facilitate users who wish to contribute to our repo, that they could run the e2e tests locally before submitting PRs. 
How do we plan to address that? I do support removing this file (and thus remove this maintenance overhead for us), maybe we could add a link to our resource generation script where we load the env var to keyvault, and users could use that as a guide to discover the env variables that are required for E2E tests?
https://github.com/Azure/azure-iot-sdk-csharp/blob/0ec02be9ac6fa9c97485a77099bdd822907a021d/e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1#L581-L622

Also, on an unrelated note, I just saw that we have some shared access keys exposed on our public repo, are we sure that this isn't a security issue: https://github.com/Azure/azure-iot-sdk-csharp/blob/0ec02be9ac6fa9c97485a77099bdd822907a021d/e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1#L618?

Reply:
1) The readme.md was talking about all the env variables we use in the tests and was thinking that would be sufficient. But it would be good to point customers to the scripts like you suggested. Let me see what is the best place in the readme.md to point to that.
2) The invalid certs are not a security issue as they are incorrect and won't work. We don't seem to expose any real shared access keys.

Reply:
This also brings a good question of anyone who uses linux or mac and not windows. Our script is only for windows powershell. The template itself might not explain how to set it up though.

Reply:
Powershell exists on Linux. Are you saying we've written it in a way that won't work for Linux, or something else?

Reply:
I've never tried it. And I was not aware linux and mac users can use it. Good to know! I'll work on updating the readme to point to that then. Otherwise we have too many things out of sync already in terms of e2e setup readme.

Reply:
I have rewritten the readme. Would appreciate feedback.